packages/create-sitecore-jss/src/templates/nextjs-sxa/package.json

```diff
@@ -1,5 +1,9 @@
 {
   "dependencies": {
-    "bootstrap": "^5.1.3"
+    "bootstrap": "^5.1.3",
+    "request": "^2.88.2"
```

Comment:
Here is an issue with `request` library, it's deprecated and we can't use it, should be replaced by another library. They don't add changes and don't fix security vulnerabilities